src/test/java/io/bf2/kafka/authorizer/CustomAclAuthorizerTest.java

```diff
@@ -310,6 +396,18 @@ private static boolean isFailedWithError(AclCreateResult aclCreateResult, String
         return aclCreateResult.exception().isPresent() && error.equals(aclCreateResult.exception().get().getMessage());
     }
 
+    private static AclBindingFilter principalFilter(String principal) {
+        AccessControlEntryFilter principalFilter = new AccessControlEntryFilter(principal, null, AclOperation.ANY, AclPermissionType.ANY);
+        return new AclBindingFilter(ResourcePatternFilter.ANY, principalFilter);
+    }
+
+    private static void assertUserPrincipalAllowedToDescribeTopic(CustomAclAuthorizer auth, String principal, String topic) {
+        AuthorizableRequestContext rc = createMockRequestContext("security-9095", KafkaPrincipal.USER_TYPE, principal, ApiKeys.DESCRIBE_CONFIGS.id);
+        Action action = new Action(AclOperation.DESCRIBE, new ResourcePattern(ResourceType.TOPIC, topic, PatternType.LITERAL), 0, true, true);
+        List<AuthorizationResult> results = auth.authorize(rc, Arrays.asList(action));
+        assertEquals(1, results.size());
```

Comment:
Should this be an `assertThat(results, hasAtLeastElement(1))`?

Reply:
It should contain a result per input in the `Arrays.asList(action)` so in this context we know there should be a single result.

Reply:
Ok.